packages/cli-kit/src/public/node/api/business-platform.ts

```diff
@@ -61,29 +61,65 @@ export async function businessPlatformRequestDoc<TResult, TVariables extends Var
   })
 }
 
+/**
+ * Sets up the request to the Business Platform Organizations API.
+ *
+ * @param token - Business Platform token.
+ * @param organizationId - Organization ID as a numeric (non-GID) value.
+ */
+async function setupOrganizationsRequest(token: string, organizationId: string) {
+  const api = 'BusinessPlatform'
+  const fqdn = await businessPlatformFqdn()
+  const url = `https://${fqdn}/organizations/api/unstable/organization/${organizationId}/graphql`
+  return {
+    token,
+    api,
+    url,
+    responseOptions: {onResponse: handleDeprecations},
+  }
+}
+
 /**
  * Executes a GraphQL query against the Business Platform Organizations API.
  *
  * @param query - GraphQL query to execute.
  * @param token - Business Platform token.
+ * @param organizationId - Organization ID as a numeric (non-GID) value.
+ * @param variables - GraphQL variables to pass to the query.
+ * @returns The response of the query of generic type <T>.
+ */
+export async function businessPlatformOrganizationsRequest<T>(
+  query: string,
+  token: string,
+  organizationId: string,
+  variables?: GraphQLVariables,
+): Promise<T> {
+  return graphqlRequest<T>({
+    query,
+    ...(await setupOrganizationsRequest(token, organizationId)),
+    variables,
+    responseOptions: {onResponse: handleDeprecations},
+  })
+}
+
+/**
+ * Executes a GraphQL query against the Business Platform Organizations API. Uses typed documents.
+ *
+ * @param query - GraphQL query to execute.
+ * @param token - Business Platform token.
  * @param organizationId - Organization ID as a numeric value.
  * @param variables - GraphQL variables to pass to the query.
  * @returns The response of the query of generic type <T>.
  */
-export async function businessPlatformOrganizationsRequest<TResult>(
+export async function businessPlatformOrganizationsRequestDoc<TResult>(
```

Comment:
I'm a bit confused. Why do we need two functions to run the query? Is this related to the PR?